qutebrowser/config/configdata.yml

```diff
@@ -1348,12 +1348,27 @@ tabs.mousewheel_switching:
 tabs.new_position.related:
   default: next
   type: NewTabPosition
-  desc: Position of new tabs opened from another tab.
+  desc: >-
+    Position of new tabs opened from another tab.
+
+    See `tabs.new_position.stacking` for controlling stacking behavior.
 
 tabs.new_position.unrelated:
   default: last
   type: NewTabPosition
-  desc: "Position of new tabs which aren't opened from another tab."
+  desc: >-
+    Position of new tabs which are not opened from another tab.
+
+    See `tabs.new_position.stacking` for controlling stacking behavior.
+
+tabs.new_position.stacking:
+  default: true
+  type: Bool
+  desc: >-
+    Stack relative tabs on top of each other when opened consecutively.
+
+    Only applies `next` and `prev` values of `tabs.new_position.related` and
```

Comment:
It does not seem to work for `prev`. I think you need to increment `_tab_insert_idx_left`. Or just make it always be `currentIndex()` for consistency.

Reply:
yup, thanks for noticing that. For some reason, I thought that the desired behavior was happening on the defaults, but I think I just didn't set the variable that time :P